README.md

```diff
@@ -198,6 +198,9 @@ You can also load your JSONified config in from an ENV variable (e.g. `KUBE_CONF
 ```
 Kubeclient::Config.new(JSON.parse(ENV['KUBE_CONFIG']), nil)
 ```
+###Supported kubernetes versions
+
+For 1.1 only the core api is supported, apis/extensions are supported in later versions.
```

Comment:
It should be in the mapping AFAIK. Since we have to manually test, review and add tests for everything we support I would rather we scope this change only for core api|oapi.

We would also need to check if we have cases of different apis using same entities.

@zakiva 
- we are supporting v1 only right? please state that here
- can you figure out with @cben which is the currect k8s version to use here?